data/hai/hai outfits.txt

```diff
@@ -188,9 +188,9 @@ outfit "Ionic Turret Prototype"
 		"hit force" 60
 		"shield damage" 100
 		"hull damage" 50
-		"ion damage" 14
-		"scrambling damage" 14
-	description "This weapon improves upon the ionization capabilities of the Ion Cannon, while also being turret-mounted, enabling it to fire at enemies in any direction, not just those directly ahead."
+		"ion damage" 22
```

Comment:
I dont mind this but its crazily too good, just re-use the values from the campaign, they feel fair (14 ion and 10 scrambling, I think)

Reply:
```suggestion
		"ion damage" 17.5
```